GNUmakefile

```diff
@@ -118,14 +118,14 @@ test_need_operator_assets: ## Add blank file in web assets if operator ui has no
 .PHONY: test_smoke
 test_smoke: test_need_operator_assets ## Run all integration smoke tests, using only simulated networks, default behavior
 	ACK_GINKGO_DEPRECATIONS=2.5.0 ginkgo -v -r --junit-report=tests-smoke-report.xml \
-	--keep-going --trace --randomize-all --randomize-suites \
+	--keep-going --race--trace --randomize-all --randomize-suites \
```

Comment:
Maybe a typo - there is no space after `--race` and before `--trace`

Reply:
Oh yeah - I biffed that rebase :facepalm: